src/elements/content-explorer/__tests__/ContentExplorer-test.js

```diff
@@ -39,4 +40,18 @@ describe('elements/content-explorer/ContentExplorer', () => {
             expect(instance.fetchFolder).toHaveBeenCalledWith('123', false);
         });
     });
+
+    describe('changeViewMode()', () => {
+        test('should initially be list view', () => {
+            const wrapper = getWrapper({});
```

Comment:
You could update `getWrapper`'s `props` param to default to an empty object to avoid passing one each time.